cli/tests/unit/fetch_test.ts

```diff
@@ -1828,3 +1829,48 @@ Deno.test(
     assertEquals(req2.headers.get("x-foo"), "bar");
   },
 );
+
+Deno.test(
+  { permissions: { net: true } },
+  async function fetchRequestBodyErrorCatchable() {
+    const listener = Deno.listen({ hostname: "127.0.0.1", port: 4514 });
+    const server = (async () => {
+      const conn = await listener.accept();
+      listener.close();
+      const buf = new Uint8Array(160);
+      const n = await conn.read(buf);
+      assertEquals(n, 160);
+      const buf2 = new Uint8Array(160);
+      const n2 = await conn.read(buf2);
+      assertEquals(n2, 6);
```

Comment:
What's special about 160 and 6? I'm guessing that 160 is the size of the request headers and 6 is the body? This seems very brittle.